test/rssfeed.cpp

```diff
@@ -3,11 +3,11 @@
 #include "3rd-party/catch.hpp"
 #include "cache.h"
 #include "configcontainer.h"
+#include "curlhandle.h"
 #include "feedretriever.h"
 #include "rssparser.h"
 
 #include "test_helpers/envvar.h"
-#include "test_helpers/stringmaker/optional.h"
```

Comment:
Might need to add this again.
The Vim plugin I'm using for C++ highlights this because it is not directly used.
However, it is used indirectly by Catch2 to convert `nonstd::optional<...>` types to string.

Reply:
I will update in a bit.
I have to rebase anyway when https://github.com/newsboat/newsboat/pull/2755 is merged (to resolve build issue)